scripts/getdata.py

```diff
@@ -0,0 +1,6 @@
+from ruvnl_consumer_app.app import fetch_data
```

Comment:
find to add this, perhaps put a descirption at the top 
"""  This is a scratch file, that is useful to run the `fetch_data`"""